crates/chain-config/src/config.rs

```diff
@@ -14,52 +14,31 @@ pub use state::*;
 
 #[cfg(test)]
 mod tests {
-    use fuel_core_types::{
-        blockchain::primitives::DaBlockHeight,
-        fuel_asm::op,
-        fuel_tx::{
-            TxPointer,
-            UtxoId,
-        },
-        fuel_types::{
-            AssetId,
-            Bytes32,
-        },
-        fuel_vm::Contract,
-    };
-    use rand::{
-        prelude::StdRng,
-        Rng,
-        RngCore,
-        SeedableRng,
-    };
     #[cfg(feature = "std")]
     use std::{
         env::temp_dir,
         fs::write,
         path::PathBuf,
     };
 
-    use super::{
-        chain::ChainConfig,
-        coin::CoinConfig,
-        contract::ContractConfig,
-        message::MessageConfig,
-        state::StateConfig,
+    use rand::{
+        rngs::StdRng,
+        RngCore,
+        SeedableRng,
     };
 
+    use super::chain::ChainConfig;
+
     #[cfg(feature = "std")]
     #[test]
-    fn from_str_loads_from_file() {
+    fn from_file_loads_from_file() {
```

Comment:
I wonder if we should be testing the writing code too. That functionality is just written into the cli endpoint rather than having its own helper like `load_from_file`. 

It would be good to have a `create_config_file` rather than testing against the `serde` and `write`.